app/components/ChapterList.js

```diff
@@ -62,6 +63,9 @@ const styles = StyleSheet.create({
     paddingRight: 20,
     backgroundColor: theme.menuBackground,
   },
+  containerContent: {
+    paddingBottom: 20,
```

Comment:
you can actually move paddingVertical:12 from the container to containerContent, in place of paddingBottom: 20